docs/meetings/frontend/2021-06-30.md

```diff
@@ -0,0 +1,26 @@
+# Frontend meeting 30 June 2021
+
+**Present:** Francisco Jesus Aapeli Lucas Darren
```

Comment:
Want to sort this and add punctuation?

Reply:
It's in "Order of Lucas' zoom window"

Reply:
Commas though?

Reply:
Hey wait a minute, you just always get to go first with your "aaaaaaaapeli"

Reply:
Anyway, it doesn't matter

Reply:
> Hey wait a minute, you just always get to go first with your "aaaaaaaapeli"

You've figured it out ;)